### PR TITLE
feat(registry): add deprecated banner

### PR DESCRIPTION
**Pull request by meszarosdezso**
Ref #1964 

I think there should also be a note somewhere about this, could someone point me to a documentation where this would fit?

### CODE REVIEW

**Comment by meszarosdezso** [comment]
An example can be visited here since there is no module currently using this directive: https://deno-website2-lmr0yi8c6-denoland.vercel.app/

( I removed the example from the PR )

File: util/registry_utils.test.ts [hunk 1]
```diff
@@ -238,3 +239,15 @@ test("parseQuery", () => {
     path: "/mod.ts",
   });
 });
+
+test("parseDeprecatedDirective", () => {
```
Comment (kt3k) [inline util/registry_utils.test.ts:243]: 👍

**Review by kt3k — APPROVED** [review]
LGTM

**Comment by meszarosdezso** [comment]
Hey @kt3k, I think this can be closed for now, there were not much response from the community on the issue and this has many conflicts now.

I would happily get back to work on it when there's a final design decision on the feature!

Closing until then.

**Pull request closed without merging**